### PR TITLE
Add nativefiledialog as a git submodule

### PR DESCRIPTION
**Pull request by DenisKolodin**
This PR uses **nativefiledialog** as submodule. I keep `build.rs`. But use `devel` branch. It's in active development. I'll change dependency to `master` when `devel` will be merged, because it's one still contains changes we need.

We can use `devel` or wait for `master`. Any option available. But PR is ready to check.

### CODE REVIEW

**Comment by DenisKolodin** [comment]
My AppVeyor to check: https://ci.appveyor.com/project/DenisKolodin/nfd-rs
MinGW and VS versions were build successfully.

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "nfd"
-version = "0.0.2"
-authors = ["Saurav Sachidanand <sauravsachidanand@gmail.com>", "Daniel Collin <daniel@collin>"]
+version = "0.0.3"
+authors = [
+	"Saurav Sachidanand <sauravsachidanand@gmail.com>",
+	"Daniel Collin <daniel@collin>",
```
Comment (emoon) [inline Cargo.toml]: I think .com is missing in my email. Not the end of the world but still :)

Comment (DenisKolodin) [inline Cargo.toml]: Ok, I'll fix it )

**Comment by DenisKolodin** [comment]
@emoon I've fixed your mail. Check, please, Is is right?

**Comment by emoon** [comment]
Looks good :)

**Comment by saurvs** [comment]
The devel branch has been merged into master.

**Comment by DenisKolodin** [comment]
That's good news! I'll update this PR.

**Comment by DenisKolodin** [comment]
I've updated this PR fully: made a new fork and added that chages again. I did it to keep submodules clean.

Building in progress: https://ci.appveyor.com/project/DenisKolodin/nfd-rs

**Comment by DenisKolodin** [comment]
It's good if you can check it locally before merge. I'm checking for linux now...

**Comment by DenisKolodin** [comment]
I checked PR with **Fedora 24** and **Windows 10** and examples. It works fine!
No other OS I can check, but I feel it will work on **Mac** too, because `nativefiledialog` hasn't major changes. Build system only, but we use own `build.rs`.

**Comment by saurvs** [comment]
This PR works fine on OS X El Capitan too. Thanks for the contribution!

**Comment by emoon** [comment]
👍

**Pull request merged**